api/src/main/java/org/apache/brooklyn/api/mgmt/rebind/mementos/Memento.java

```diff
@@ -77,7 +79,8 @@
 
     public Collection<Object> getTags();
     
+    Map<String, Set<BrooklynObject>> getRelations();
```

Comment:
Agree, string is better for consistency right now.  It means we need untyped `lookup(...)` methods during rebind, which I've added (simply pass `null` as type constraint argument).

Using `BrooklynObject` is perhaps nicer if we do that everywhere (and clean up refs to `entityProxy` instead of `entity` etc in the generated XML); however if we switch to a yaml representation here which I'd like to eventually then it's a moot point.